Assignments/discord/KCreel/persiflage/src/App.js

```diff
@@ -0,0 +1,163 @@
+import React, {useState} from 'react';
+import './App.css';
+import { Grid, Container, List } from 'semantic-ui-react';
+
+const uuidv1 = require('uuid/v1');
+//[items, setItem] = useState(
+const Server = ({ servers, setCurrentServer }) => {
+  return (
+    <Grid.Column>
+     {servers.map(server =>(
+       <Grid.Row>
+         <span onClick={changeServer(setCurrentServer, server.serverName)}>{server.serverName}</span>
```

Comment:
As soon as the page loads this will execute and from looking at it, I dont see it return a function...so I think this is an accident perhaps
```
<span onClick={() => {changeServer(setCurrentServer, server.serverName)} }>{server.serverName}</span>
```